docs/source/users/options/fitting_option.rst

```diff
@@ -134,8 +134,6 @@ Current Hessian methods are:
 
 * ``default`` - Hessian information is not passed to minimizers
 * ``analytic`` - uses the analytic Hessian extracted from the fitting problem.
-* ``scipy`` -  uses :ref:`SciPy's finite difference approximations <scipy-hes>`.
-* ``numdifftools`` - uses the python package :ref:`numdifftools <numdifftools-hes>`.
```

Comment:
I can't recreate the issue. I don't think we should remove this until we can confirm it's not just a test on an outdated version. Can you run it?

Reply:
Looks like you are right, thanks for finding that 👍 I'll add it back in

Reply:
It runs fine for me too